spec/ar_lazy_preload/preload_associations_lazily_spec.rb

```diff
@@ -138,7 +138,7 @@
       expect do
         subject.flat_map do |u|
           u.posts.where(
-            created_at: ::Time.zone.at(0)..(::Time.zone.at(0) + 1)
+            created_at: ::Time.at(0).utc..(::Time.at(0).utc + 1),
```

Comment:
I explain this in the commit message, but the gist here is that `Time.zone` returns `nil` in non-Rails contexts because the time zone is not automatically set on app start. I think this achieves the same result without using a pre-set zone, but if you prefer this can also be solved by setting the time zone at the beginning of the test.